### PR TITLE
loops in js

### PR DESCRIPTION
**Pull request by anascence**

### CODE REVIEW

File: docs/loops.md [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+```
+
+### Example:
+
+```jsx
+//create object
+const person = {
+	name: "Tawan",
+	age: 20,
+	occupation: "GOD"
```
Comment (Lxkas) [inline docs/loops.md:242]: ???????😱

Comment (chinathaip) [reply docs/loops.md:242]: 5555555555555555555555555555555555

Comment (anascence) [reply docs/loops.md:242]: 555555

**Comment by chinathaip** [comment]
@NayHtetKyaw  LGTM!

maybe @Lxkas might have something to share us? or else let's merge

**Comment by Lxkas** [comment]
One sec don't merge yet

**Comment by chinathaip** [comment]
this guy comments with one account, then add more commit with another

**Comment by anascence** [comment]
> One sec don't merge yet

also in readme file

is the command npm build should be -> npm run build ??

**Comment by anascence** [comment]
> this guy comments with one account, then add more commit with another

god behaviors

**Comment by Lxkas** [comment]
> this guy comments with one account, then add more commit with another

bro I forgor to change account XD
I work on a lot of her projects and she does on mine so it kinda makes sense to set defaults

**Comment by Lxkas** [comment]
@NayHtetKyaw @chinathaip give it one more review, if everything passes your standards go ahead and merge, I'm going to sleep

**Comment by chinathaip** [comment]
sorry, just wanted mine to have similar headings XDDD

**Review by chinathaip — APPROVED** [review]
LGTM! I really like the task at the end, should be challenging enough for them!

**Pull request merged**